lib/package/commands.coffee

```diff
@@ -28,6 +28,11 @@ module.exports =
         @withInk ->
           boot()
           juno.runtime.evaluation.evalAll()
+      'julia-client:run-all-chunks': (event) =>
+          cancelComplete event
+          @withInk ->
```

Comment:
The following lines need to be indented here, to be part of the `@withInk` callback